internal/supervisor/installation.go

```diff
@@ -1244,11 +1244,11 @@ func (s *InstallationSupervisor) finalDeletionCleanup(installation *model.Instal
 		if record.DeleteAt > 0 {
 			continue
 		}
-		err = s.dnsProvider.DeleteDNSRecords([]string{record.DomainName}, logger)
-		if err != nil {
-			logger.WithError(err).Error("Failed to delete DNS record from Cloudflare")
-			return model.InstallationStateDeletionFinalCleanup
-		}
+		// err = s.dnsProvider.DeleteDNSRecords([]string{record.DomainName}, logger)
```

Comment:
This is a oversight on my part, had a lot of things stashed and this slipped in. I commented out that code on my first week because I had a corrupted state, since I had installations pending deletion trying to delete DNS Records but I still had no access. This allowed me to safely pass that post deletion step, but slipped in this PR. Thanks a lot for catching it!